cmd/influx_inspect/export.go

```diff
@@ -99,19 +106,25 @@ func (c *cmdExport) writeFiles() error {
 		w = gzip.NewWriter(w)
 	}
 
-	// Write out all the DDL
-	fmt.Fprintln(w, "# DDL")
-	for key, _ := range c.files {
-		keys := strings.Split(key, string(byte(os.PathSeparator)))
-		fmt.Fprintf(w, "CREATE DATABASE %s\n", keys[0])
-		fmt.Fprintf(w, "CREATE RETENTION POLICY %s ON %s DURATION inf REPLICATION 1\n", keys[1], keys[0])
+	if c.tsmDir == "" {
```

Comment:
`tsmdir` could be used with .tsm files sitting in any directory, It may not be possible to get the DDL when this flag is used because the DB and RP name come from the directory structure.